src/helpers/createProject.ts

```diff
@@ -5,18 +5,21 @@ import { installPackages } from "./installPackages.js";
 import { scaffoldProject } from "./scaffoldProject.js";
 import { selectAppFile, selectIndexFile } from "./selectBoilerplate.js";
 
-export const createProject = async (
-  projectName: string,
-  packages: PkgInstallerMap,
-) => {
+export const createProject = async (opts: {
```

Comment:
Could you extract this type like this:

```tsx
interface CreateProjectProps {
    opts: {
         projectName: stirng;
         ....
    }
}
```

I usually put the interfaces/types just right above the function.